code/game/machinery/camera/camera.dm

```diff
@@ -85,7 +85,7 @@
 
 	var/list/datum/weakref/localMotionTargets = list()
 	var/detectTime = 0
-	var/area/station/ai_monitored/area_motion = null
+	var/datum/motion_group/area_motion = null
```

Comment:
Do we need reference handing for this, like setting it to `null` in destroy or w/e